pyerrors/input/pandas.py

```diff
@@ -142,9 +138,9 @@ def _serialize_df(df, gz=False):
         serialize = _need_to_serialize(out[column])
 
         if serialize is True:
-            out[column] = out[column].transform(lambda x: create_json_string(x, indent=0)if x is not None else None)
+            out[column] = out[column].transform(lambda x: create_json_string(x, indent=0) if x is not None else None)
             if gz is True:
-                out[column] = out[column].transform(lambda x: gzip.compress((x).encode('utf-8')))
+                out[column] = out[column].transform(lambda x: gzip.compress((x if x is not None else '').encode('utf-8')))
```

Comment:
Thanks, I think this is a cleaner solution! I didn't fully follow your discussion, did you agree to serialize `None` as an empty string?

Reply:
At least, this is the default behavior of `pandas.to_csv`. The more important point was to settle on how to interpret empty strings on input.

Reply:
> Hi. In the current version, an error would be thrown if someone would try to export a `NaN` (not a Nan-Obs). So for the moment, as long as nobody has a problem because they would like to export these `nan`s, you could assume that empty strings (using `keep_default_na=False`) can be set to `None` after the import, because nothing else could have been exported.

I am currently testing this... but there is no such error, at least after the changes of this branch. See e.g. the output of:
`
my_dict = {"int": 1,
           "float": -0.01,
           "Obs1": pe.pseudo_Obs(87, 21, "test_ensemble"),
           "Obs2": pe.pseudo_Obs(-87, 21, "test_ensemble2")}
my_df = pd.DataFrame([my_dict] * 4)
my_df.loc[1, "int"] = np.nan
my_df.loc[0, "Obs1"] = None
my_df.loc[2, "Obs1"] = None
pe.input.pandas.dump_df(my_df, 'df_output', gz=True)
`
I would suggest to put a warning in `_serialize_df`. Or is there a better way to handle this?